spec/integration/post_show_spec.rb

```diff
@@ -17,39 +17,31 @@
     @post5 = Post.create(title: 'Automobile engineering', text: 'Reading', likes_counter: 0,
                          comments_counter: 0, author_id: @user2.id)
     Comment.create(text: 'This is great', post_id: @post1.id, author_id: @user2.id)
+    visit post_path(@user1.id, @post1.id)
   end
 
   context 'Post index page' do
-    it 'Should see the user name' do
-      visit post_path(@user1.id, @post1.id)
+    it 'Should see the post title' do
       expect(page).to have_content('This is a post')
-      expect(page).to have_content('Accounting made easy')
     end
 
-    it 'Should see number of posts' do
-      visit post_path(@user1.id, @post1.id)
+    it 'Should see number of comments' do
       expect(page).to have_content('Comments')
     end
 
-    it 'Should see the users first three posts' do
-      visit post_path(@user1.id, @post1.id)
-      expect(page).to have_content('This is a post')
-      expect(page).to have_content('Accounting made easy')
+    it 'Should see number of likes' do
+      expect(page).to have_content('Likes')
     end
```

Comment:
**[OPTIONAL]**

- [ ] It would be very great if you test your comments and likes counter properly :+1: because now you are just testing if these keywords are on your page :blush: But they might be buttons or letters which are on your page so it would be great if you be very specific while counters in this specific case :+1: You can make your tests as shown below to make them specific to the counters :+1: :arrow_down:

```ruby
    it 'Should see number of comments' do
      expect(page).to have_content('Comments(1)')
    end

    it 'Should see number of likes' do
      expect(page).to have_content('Likes(0)')
    end
```